doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -4,4 +4,218 @@
+-----------------------------------
+
+When code managers receive a pull request to commit the code changes, it is recommended that they add at least two code reviewers to review the code and at least one of the reviewers has write permission. The reviewers will write comments about the code changes and give a recommendation as to whether the code changes can be committed. What kinds of code changes will be accepted in the repository is beyond the scope of this document; future ufs-weather-model code management documents may have a detailed answer for that.
+
+Reviewers may suggest some code changes during the review process. Developers need to respond to these comments in order to get code changes committed. If developers make further changes to their branch, reviewers need to check the code changes again. When both reviewers give recommendation to commit the code, code managers will merge the changes into the repository.
+
+----------------------------
+Conducting regression tests
+----------------------------
+
+Only developers using Tier 1 and 2 platforms can run the ufs-weather-model regression tests. Other developers need to work with the code managers to assure completion of the regression tests.
+
+
+To run regression test using rt.sh
+
+rt.sh is a bash shell file to run the RT and has the following options:
+
+.. code-block:: console
+
+   Usage: $0 -c <model> | -f | -s | -l <file> | -m | -r | -e | -h
+   -c create new baseline results for <model>
+   -f run full suite of regression tests
+   -s run standard suite of regression tests
+   -l run test specified in <file>
+   -m compare against new baseline results
+   -r use Rocoto workflow manager
+   -e use ecFlow workflow manager
+   -h display this help
+
+.. code-block:: console
+
+   % cd ufs-weather-model/tests
+   % ./rt.sh -f
+
+This command can only be used on a NOAA machine using the Intel compiler, where an official baseline is available. For information on testing the CCPP code, or using alternate computational platforms, see the following sections.
+
+This command and all others below produce log output in ./tests/log_machine.compiler. These log files contain information on the location of the run directories that can be used as templates for the user. Each rt*.conf contains one or more compile commands preceding a number of tests.
+
+Regression test log files (ufs-weather-model/tests/Compile_$(MACHINE_ID).log and ufs-weather-model/tests/RegressionTests_$(MACHINE_ID).log ) will be updated.
+
+Regression testing is only possible on machines for which baselines exist, that is Tier 1 and 2 platforms.
+
+When porting the code to a new machine, it is useful to start by establishing a personal baseline. Future runs of the RT can then be compared against the personal baseline to ascertain that the results have not been inadvertently affected by code developments. The rt.sh -c option is used to create a personal baseline.
+
+./rt.sh -l rt.conf -c fv3 # create own reg. test baseline
```

Comment:
Remove ``fv3`` after ``-c``

Reply:
Yes, will do. My job was not to _work_ on the documentation but to merge what is there now. Hmpf. I will correct it.